feat: 71 feature make all bash commands default to exec (#74)
grafana/killercoda: docs/transformer.md

<!-- INTERACTIVE exec END -->
```

> #### NOTE
>

> > By default, the tool defaults to make code fenced with `bash` as executable. Meaning you do not need to specify the `<!-- INTERACTIVE exec START/STOP -->` directives for bash code blocks. It is possible to override this behaviour by adding the directive such as `<!-- INTERACTIVE copy START/STOP -->` to the fenced code block as directives always take precedence.

Reviewer: ```suggestion
> By default, the tool makes `bash` fenced code blocks executable so you don't need `<!-- INTERACTIVE exec START/STOP -->` directives for bash code blocks.
> You can override this behavior with the `<!-- INTERACTIVE copy START/STOP -->` directives which take precedence over the default behavior.
```

> > #### NOTE
> >
> > By default, the tool defaults to make code fenced with `bash` as executable. Meaning you do not need to specify the `<!-- INTERACTIVE exec START/STOP -->` directives for bash code blocks. It is possible to override this behaviour by adding the directive such as `<!-- INTERACTIVE copy START/STOP -->` to the fenced code block as directives always take precedence.

Reviewer: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

```suggestion
> [!NOTE]
> By default, the tool makes `bash` fenced code blocks executable so you don't need `<!-- INTERACTIVE exec START/STOP -->` directives for bash code blocks.
> You can override this behavior with the `<!-- INTERACTIVE copy START/STOP -->` directives which take precedence over the default behavior.
```

> 
> 
> 

Reviewer: ```suggestion

```

<!-- prettier-ignore-end -->

### Copy

Copy directives tell the transform tool to make the contained fenced code block copyable.

The start marker is:

```markdown
<!-- INTERACTIVE copy START -->
```

The end marker is:

```markdown
<!-- INTERACTIVE copy END -->
```

> > #### NOTE
> >
> > By default, the tool defaults to make all fenced code apart from `bash` as copyable. Meaning you do not need to specify the `<!-- INTERACTIVE copy START/STOP -->` directives for code blocks. The primary reason for using copy directives is to override the default behaviour for `bash` code blocks.

Reviewer: ```suggestion
> [!NOTE]
> By default, the tool makes all fenced code blocks other than `bash` copyable so you don't need `<!-- INTERACTIVE copy START/STOP -->` directives for those code blocks.
> You can override this behavior with the `<!-- INTERACTIVE exec START/STOP -->` directives which take precedence over the default behavior.
```
